Makefile

```diff
@@ -171,7 +171,7 @@ analyze: $(JS)
 		echo "ccflags-y += -DZJS_SNAPSHOT_BUILD" >> src/Makefile; \
 	fi
 ifeq ($(DEV), ashell)
-	@cat fragments/prj.mdef.dev >> prj.mdef
+	@cat fragments/prj.mdef.ashell >> prj.mdef
```

Comment:
They're in different sections, I don't want to risk that right now. The mdef stuff is in one place, the prj.conf is in one place.